prov/shm/src/smr_ep.c

```diff
@@ -43,6 +43,10 @@
 #include "smr.h"
 #include "smr_dsa.h"
 
+#if HAVE_XPMEM
+#include "ofi_xpmem.h"
+#endif
```

Comment:
the ofi_xpmem header already have a check within it so you shouldn't need to exclude the include here